runtime.go

```diff
@@ -1945,7 +1869,8 @@ func (r *Runtime) toValue(i interface{}, origValue reflect.Value) Value {
 		// all its methods. This is done deliberately instead of converting it to a `Date` because these two types are not fully
 		// compatible: `time.Time` includes zone, whereas JS `Date` doesn't. Doing the conversion implicitly therefore would
 		// result in a loss of information.
-		return r.newDateObject(i, true, r.global.DatePrototype)
+		// todo guessing that I need to use the template object instead
```

Comment:
This is one of our changes that don't exist in upstream, was just a comment that I'm assuming this should be the case.